src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -258,7 +259,7 @@ export default function TokenDetails({
           <TokenDetailsSkeleton />
         )}
 
-        <RightPanel onClick={() => isBlockedToken && setOpenTokenSafetyModal(true)}>
+        <RightPanel isInfoTDPEnabled onClick={() => isBlockedToken && setOpenTokenSafetyModal(true)}>
```

Comment:
```suggestion
        <RightPanel isInfoTDPEnabled={isInfoTDPEnabled} onClick={() => isBlockedToken && setOpenTokenSafetyModal(true)}>
```

?

Reply:
because `isInfoTDPEnabled` is a boolean? with the same name, the component can intake the parameter directly without needing to be wrapped. I find this syntax to be more succinct without a sacrifice to readability

Reply:
is this true? afaiu if it's just `isInfoTDPEnabled`, it's passed in as true.

Reply:
I was 100% mistaken and you were both 100% correct